polaris-tokens/scripts/index.ts

```diff
@@ -1,8 +1,9 @@
-import {osColorSchemes, tokens} from '../src';
+import {tokens} from '../src';
 
+import {toTokenValues} from './toTokenValues';
 import {toJSON} from './toJSON';
 import {toStyleSheet} from './toStyleSheet';
 
 (async () => {
-  await Promise.all([toJSON(tokens), toStyleSheet(tokens, osColorSchemes)]);
+  await Promise.all([toTokenValues(), toJSON(tokens), toStyleSheet(tokens)]);
```

Comment:
```suggestion
  await Promise.all([toTokenValues(tokens), toJSON(tokens), toStyleSheet(tokens)]);
```

Should `toTokenValues` also accept a reference to the `tokens` object?